src/lesson3/_Cikl/Task3.java

```diff
@@ -0,0 +1,15 @@
+package lesson3._Cikl;
+
+public class Task3 {
+    public static void main(String[] args) {
+
+        int g = 7;
+        while (g <= 99) {
+            System.out.println(g);
+            g = g + 7;
```

Comment:
Is it working now?

Reply:
yeas, it's working.
![Task3](https://github.com/NEULOVIM/homeWork/assets/161806973/39d18a8f-5c31-4a7a-9580-2480c781d8b6)